### ISSUE TITLE
Fonts not loading correctly

### ISSUE THREAD

**Issue by sondrelg** [Kozea/WeasyPrint#1333]
Hi! 

I have this HTML that I want to write to a PDF:

```html
<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins:300,400,500,600,700">

<p style="font-family: Poppins">Test</p>
```

As far as I can tell it's valid, and the font loads fine when run in a browser. Unfortunately, the fonts do not load when generating PDFs using the same HTML.

We've tested this five places, and it mysteriously works on one macbook and a linux, and fails on two other macbooks and our Heroku dynos, none of which have Poppins installed locally. If you install Poppins locally it works, but that doesn't seem like it should be necessary.

### How to reproduce

```python
import pikepdf
import requests
from weasyprint import HTML, CSS
from weasyprint.fonts import FontConfiguration

response = requests.get(f'https://fonts.googleapis.com/css?family=Poppins:300,400,500,600,700')
css_font_declarations = response.content.decode('utf-8')

font_config = FontConfiguration()
html = HTML(string='<p style="font-family: Poppins">Test</p>')
css = CSS(string=css_font_declarations, font_config=font_config)
html.write_pdf('/tmp/example.pdf', stylesheets=[css], font_config=font_config)

# This inspects the generated PDF
pdf = pikepdf.open('/tmp/example.pdf')
for page in pdf.trailer['/Root']['/Pages']['/Kids']:
    # print(repr(page))
    assert 'Poppins' in repr(page)
```

When printing the page, this is my output:

```< pikepdf.Dictionary(Type="/Page")({
  ...
      "/Font": {
          "/f-0-0": {
              /BaseFont": "/FZNKSN+Verdana",
              "/Encoding": "/WinAnsiEncoding",
              "/FirstChar": 32,
              "/FontDescriptor": {
              "/Ascent": 1005,
              "/CapHeight": 1000,
              "/Descent": -209,
              "/Flags": 32,
              "/FontBBox": [-494, -303, 1446, 1000],
              "/FontFamily": "Verdana",
              "/FontFile2": pikepdf.Stream(stream_dict={
                  "/Filter": "/FlateDecode",
                  "/Length": 2651,
                  "/Length1": 3964
               }, data= < ... >),
              "/FontName": "/FZNKSN+Verdana",
              "/ItalicAngle": 0,
              "/StemH": 80,
              "/StemV": 80,
              "/Type": "/FontDescriptor"
          },
          "/LastChar": 116,
          "/Subtype": "/TrueType",
          "/ToUnicode": pikepdf.Stream(stream_dict={
              "/Filter": "/FlateDecode",
              "/Length": 242
          }, data= < ... >),
          "/Type": "/Font",
          "/Widths": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
                      0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 616, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
                      595, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 520, 394]
         
      ...
```

When generating the PDF, the results are the same, so it doesn't seem to be a problem with how we're checking it.

**Comment by liZe** [comment]
Hello!

That’s probably a duplicate of #1318. Do you use Pango 1.48.3?

**Comment by sondrelg** [comment]
Checked two of the failing machines and they are on pango 1.48.3 and 1.48.4 it looks like 👍 

Just so I don't misinterpret anything: is the fix to downgrade pango below 1.48.2 for the current weasyprint latest version, and wait for a new weasyprint release that supports pango 1.48.3+ before upgrading? Do you have any predictions as to when the new pango version might become compatible/when you will release a fix?

Thanks so much for your help 👏

**Comment by liZe** [comment]
> Just so I don't misinterpret anything: is the fix to downgrade pango below 1.48.2 for the current weasyprint latest version, and wait for a new weasyprint release that supports pango 1.48.3+ before upgrading?

Yes, you can:
- downgrade Pango to 1.48.2, or
- apply 47043a1, or
- install the current 52.x branch.

A new release will be release soon, in a few days/weeks, before the end of the month.
